web/src/lib/utils/asset-utils.ts

```diff
@@ -527,3 +527,41 @@ export const archiveAssets = async (assets: AssetResponseDto[], archive: boolean
 export const delay = async (ms: number) => {
   return new Promise((resolve) => setTimeout(resolve, ms));
 };
+
+export const canCopyImageToClipboard = (): boolean => {
+  return !!(navigator.clipboard && window.ClipboardItem);
+};
+
+const imgToBlob = async (imageElement: HTMLImageElement) => {
+  const canvas = document.createElement('canvas');
+  const context = canvas.getContext('2d');
+
+  canvas.width = imageElement.width;
+  canvas.height = imageElement.height;
```

Comment:
This doesn't seem right. It just takes the dimensions of the image element, but we should be using the dimensions of the original image.

Reply:
This how it currently works. It always copy the image shown in the asset viewer. Do you think we should always copy the original asset (even if the asset is incompatible with the browser) ?

Reply:
This isn't about thumbnail vs original, but about the image dimensions of the copied image. The dimensions of the asset shown in the photo viewer are used for the canvas, so if you resize your browser the copied image will have different dimensions.

Reply:
Oh sorry for the confusion, it should be fixed